pkg/reconciler/openshift/tektonpipeline/extension.go

```diff
@@ -18,11 +18,21 @@ package tektonpipeline
 
 import (
 	"context"
+	"github.com/go-logr/zapr"
+	mfc "github.com/manifestival/client-go-client"
+	"github.com/tektoncd/operator/pkg/client/clientset/versioned"
+	operatorclient "github.com/tektoncd/operator/pkg/client/injection/client"
+	"go.uber.org/zap"
+	"knative.dev/pkg/injection"
+	"knative.dev/pkg/logging"
+	"os"
+	"path/filepath"
 	"strings"
 
```

Comment:
This needs to be passed through `goimports`

Reply:
done